File: c#/ingrediente.cs
public class ingrediente{
	private float _cantidad;
	private insumo _Insumo;

	public ingrediente(){}

	public ingrediente(float _cantidad, insumo _Insumo){
		this._cantidad = _cantidad;
		this._Insumo = _Insumo;
	}

	public float Catnidad{
		get{
			return _cantidad;
		}
		set{
			_cantidad = value;
		}
	}

	public insumo Insumo{
		get{
			return _Insumo;
		}
		set{
			_Insumo = value;
		}
	}
}

File: java/Ingrediente.java
public class Ingrediente{
	private int cantidad;
	private Insumo insumo;
	
	public Almacen(int cantidad,Insumo insumo){
		this.cantidad=cantidad;
		this.insumo=insumo;
	}
	
	public void setcantidad(int cantidad){
		this.cantidad=cantidad;
	}
	public int getcantidad(){
		return this.cantidad;
	}
		
	public void setinsumo(Insumo insumo){
		this.insumo=insumo;
	}
	public Insumo getinsumo(){
		return this.insumo;
	}
	
}